satellite/accounting/projectusage_test.go

```diff
@@ -40,6 +40,8 @@ func TestProjectUsageStorage(t *testing.T) {
 	testplanet.Run(t, testplanet.Config{
 		SatelliteCount: 1, StorageNodeCount: 6, UplinkCount: 1,
 	}, func(t *testing.T, ctx *testcontext.Context, planet *testplanet.Planet) {
+		planet.Satellites[0].Accounting.Tally.Loop.Pause()
```

Comment:
Had to disable it here, because live accounting has a bug: https://storjlabs.atlassian.net/browse/V3-2852